apps/microsoft-teams/frontend/src/hooks/useGetTeamsChannels.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useCallback, useState } from 'react';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { ConfigAppSDK } from '@contentful/app-sdk';
+import { AppInstallationParameters, TeamsChannel } from '@customTypes/configPage';
+
+const useGetTeamsChannels = () => {
+  const [channels, setChannels] = useState<TeamsChannel[]>([]);
+  const sdk = useSDK<ConfigAppSDK>();
```

Comment:
Ah, I was wondering why we were using the SDK in this hook and it's because we need to pass the `tenantId`. We actually might want to use the value from the parameters state in the `ConfigPage` component in case the user makes any updates to the tenant id.